web/packages/teleport/src/Roles/RoleEditor/RequiresResetToStandard.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Info } from 'design/Alert/Alert';
+import { ButtonSecondary, Text } from 'design';
+
+export const RequiresResetToStandard = ({ reset }: { reset(): void }) => (
+  <Info>
+    <Text>
+      Some fields were not readable by the standard editor. To continue editing,
+      go back to YAML editor or reset the affected fields to standard settings.
+    </Text>
```

Comment:
@avatus That was one of my (more ambitious) plans, but I'm not sure if I'll make it before v17. It's definitely a good idea in case of this editor, as it's gonna be quite complex.